test/simple/test-process-active-wraps.js

```diff
@@ -65,9 +65,12 @@ var handles = [];
   });
   function onclose() {
     if (++n === handles.length) {
-      setImmediate(function() {
-        assert.equal(process._getActiveHandles().length, 0);
-      });
+        // give server handle a chance to close - see GH #8986
```

Comment:
Do you think we could add to this comment (or at the beginning of this file, as a general comment) that this test depends a lot on the implementation of how handles are closed, and as such can and will likely break in the future? 

We could add a note saying that if the test breaks, it may be because of a correct change in the way handles are closed, and it doesn't necessarily means that node is broken.

It may save us some time next time this test breaks.